### PR TITLE
WIP - Factorize header url

### PR DESCRIPTION
**Pull request by Vampouille**
need https://github.com/georchestra/geoserver/pull/14

### CODE REVIEW

**Comment by Vampouille** [comment]
Please do not merge, we need to adapt header on GN

**Comment by Vampouille** [comment]
@fvanderbiest does I miss some webapps ?

**Comment by fvanderbiest** [comment]
analytics ? but this one will soon be deprecated so I guess that's OK.

**Comment by fvanderbiest** [comment]
geowebcache too

File: mapfishapp/src/main/webapp/WEB-INF/jsp/header.jsp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<%@ page pageEncoding="UTF-8"%>
+
+<%@ page import="org.springframework.web.context.support.WebApplicationContextUtils" %>
+<%@ page import="org.springframework.context.ApplicationContext" %>
+<%@ page import="org.springframework.web.servlet.support.RequestContextUtils" %>
+<%@ page import="org.georchestra.commons.configuration.GeorchestraConfiguration" %>
+
+<%
+
+String headerHeight = "90px";
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/WEB-INF/jsp/header.jsp]: oops, "px" here is not a good idea.

Comment (fvanderbiest) [reply mapfishapp/src/main/webapp/WEB-INF/jsp/header.jsp]: fixed with c5aa4cf

**Pull request merged**